wled00/json.cpp

```diff
@@ -487,6 +487,10 @@ bool deserializeState(JsonObject root, byte callMode, byte presetId)
     }
   }
 
+  if (root.containsKey(F("np"))) {
+    doAdvancePlaylist = root[F("np")].as<bool>(); //advances to next preset in playlist when true
```

Comment:
At first I tried it with only this line, which worked fine for JSON API but the problem is that it was overriding the HTTP API since it evaluates to false when not present. Had to wrap it in the if/contains so that it would only run this logic when the JSON actually contained "np".

Reply:
You can always use the construct from ArduinoJson:

`doAdvancePlaylist = root[”np] | doAdvancePlaylist;`

or

`CJSON(root[”np”], doAdvancePlaylist);`

as seen elsewhere.

Reply:
@blazoncek ok thanks, done and added new commit